engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -320,6 +345,7 @@ public StartContainerEvent(String containerId, String containerNodeId)
     {
       this.containerId = containerId;
       this.containerNodeId = containerNodeId;
+      this.setEventLogLevel(LogLevel.INFO);
```

Comment:
if stopContainerEvent is at WARN level shouldn't startContainerEvent be also at WARN level?